backend/src/main/java/org/ukma/spring/crooodle/dto/ReviewCreateRequestDto.java

```diff
@@ -0,0 +1,34 @@
+package org.ukma.spring.crooodle.dto;
+
+
+import jakarta.validation.constraints.*;
+
+import lombok.Builder;
+import lombok.Value;
+
+
+@Value
+@Builder
+public class ReviewCreateRequestDto {
+
+    @NotNull
+    Long hotelId;
+
+    @NotNull
+    Long userId;
```

Comment:
We should get user from auth context. You don't want user to fake their id, do you?